docs/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.md

```diff
@@ -1,23 +1,27 @@
 ---
 sort: 5
 weight: 5
-title: Alerting with vmalert and Managed VictoriaMetrics
+title: Alerting with vmalert and VictoriaMetrics Cloud
 menu:
   docs:
-    parent: "managed"
+    parent: "cloud"
     weight: 5
 aliases:
-- /managed-victoriametrics/alerting-vmalert-managed-victoria-metrics.html
+  - /victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud.html
+  - /victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud/
```

Comment:
/victoriametrics-cloud/alerting-vmalert-victoria-metrics-cloud/index.html

Reply:
and please change this in all depended docs

Reply:
updated for all victoriametrics-cloud folder